src/components/DataFetching/index.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
```

Comment:
Move this file in the `api` folder. 
Leave only ` axios.get`.
You don't need here `then` and `catch`.